content/docs/tasks/traffic-management/custom-gateway/index.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Add a new Custom Ingress Gateway
+description: Describes how to add a new custom ingress gateway manually.
+weight: 89
+keywords: [ingress,traffic-management]
+---
+
+This post provides instructions to manually create a new custom ingress [gateway](/docs/reference/config/istio.networking.v1alpha3/#Gateway) with automatic provisioning of certificates based on cert-manager.
+This task was validated on AWS Infrastructure.
+
+## Before you begin
+
+* Setup Istio by following the instructions in the
+  [Installation guide](/docs/setup/).
```

Comment:
What about cert-manager?

Reply:
I've updated with a link to cert-manager installation